axelor-supplychain/src/main/resources/domains/MrpLineType.xml

```diff
@@ -28,6 +28,14 @@
     <integer name="includeOneOffSalesSelect"
       selection="supplychain.mrp.line.off.of.sales.select" title="One-off sales"/>
 
+    <many-to-one ref="com.axelor.meta.db.MetaField" name="fieldSelect"
```

Comment:
the name should follow the naming convention, here : metaField instead of fieldSelect